### DISCUSSION TITLE
Troubles with my CPU usage and plots number, I need help?!

### DISCUSSION THREAD

**Discussion by runechronos** [swar/Swar-Chia-Plot-Manager#102] [Q&A]
Hy, 

I have some trouble finding proper settings. I was initially using my gaming PC with an **i78700 (6 cores/12threads)** with an **nVme 1To** (for tmp) and **24go of DDR4**. I was doing 3 plots at the same time (3 phases 1 simultaneously) and it was working correctly, i was doing like 3 plots in **8-10hours** (at start it was taking 8 hours, and 2 days after it was taking 10-11hours i didn't get why).

I decided to build another PC only dedicated to creating plots. With what I had. So i built it with an **i7-6700 (4 cores/8threads)** with **24go of DDR4**, and an **nVme 1To**  connected to a regular m.2 port (32bg/s max) (for tmp-plots) and it's not working properly, i don't get why. I also set to have 3 plots simultaneously like below:

```
- max_concurrent : 3
- threads : 2 (in order to have 2 remaining threads for the system win10) -> i understood it was the number of threads per plot, so as i'm doing 3 plots i put 2 here in order to have 6 in total running plots and 2 free for system)
- memory_buffer : 6000 (got it is per plot setting, as i have 24go, it lets 6go left for the system)
```

in the **"manager.py view"** it's written that around 50-65% of my cpu is used in phase 1, i got around 1.10% in 19minutes. It's awfully slow and i don't get why. I was like around 2% in 7-8min with my i7-8700. The difference can't be so huge. I got a friend with an old i7-870 with about ~40% less performance overall normally than an i7-6700, but he is doing 2 plots at the same time in about 10-12hours. So i don't get what is the behaviour on my new setup :s

What might be happening and what can I do ??? I don't get it, it should be fine with an i7-6700, it does not seem to handle 3 plots simultaneously.

I even tried to add the folders of the tmp-plots and final-plots as exception in Windows Defender, I really don't get it

Thanks in advance.

**Comment by Amneamnius** [comment]
1TB NVME can handle about 3 parallel plots, with a 4th starting once any plot goes into phase4.

After that you'll be limited by your threads, you only have 8 on the 2nd cpu so you'd want to do about 2-3 threads (3 would likely have your cpu at 100% more often). You can also stagger the plots by about 30-50m (1/4 of your plot time) to try and reduce the cpu load at any one time.

There's some slight speed increase as you go up to 4 threads, beyond that there might not be much more benefit (vs making new plots).

The time difference is likely due to the 8700 being 0.6 GHz faster. Although your friend is doing plots faster you want to look at the total number of plots you can make each day.

**Comment by runechronos** [comment]
Thanks for your answer @Amneamnius , I'm starting to think my I7 6700 is defectuous. 0.6ghz can't make me be at 11% with 2 plots after 20hours.
Do you know if your can pause plots with the scripts ? Or resume plots ? If i need to restart my computer are the current plots dead like with the gui or can I do something about it ?

**Comment by Amneamnius** [comment]
No you'd have to kill processes and delete temp files

Check nvme temps too? It can throttle due to temp. It should not be at 100% usage in task manager too
